bgpd/bgp_bmp.c

```diff
@@ -558,6 +607,28 @@ static int bmp_send_peerup(struct bmp *bmp)
 	return 0;
 }
 
+static int bmp_send_peerup_vrf(struct bmp *bmp)
+{
+	struct bmp_bgp *bmpbgp = bmp->targets->bmpbgp;
+
+	/* send unconditionally because state may has been set before the
+	 * session was up. and in this case the peer up has not been sent. */
+	bmp_bgp_update_vrf_status(bmpbgp, vrf_state_unknown);
+
+	struct stream *s = bmp_peerstate(bmpbgp->bgp->peer_self, bmpbgp->vrf_up == vrf_state_down);
+
+	if (!s) {
+		zlog_warn("bmp: peer state error");
```

Comment:
Personally, I don't understand this warning (or error)...

Reply:
removed